src/pages/Popup/components/Settings.jsx

```diff
@@ -135,11 +146,15 @@ class Settings extends Component {
       selectedSec: data['BeforeSeconds'],
       selectedSwi: data['AutoJoin'],
     });
+
     console.log(this.state);
   }
 
   render() {
     const { classes } = this.props;
+    const styleobj = {
```

Comment:
We already have `useStyles` defined. Try to shift this over there as all the styling attributes have been mentioned over there.